services/common/encryption/key_fetcher_factory.cc

```diff
@@ -86,7 +89,7 @@ std::unique_ptr<KeyFetcherManagerInterface> CreateKeyFetcherManager(
     return std::make_unique<server_common::FakeKeyFetcherManager>();
   }
 
-  google::scp::cpio::PrivateKeyVendingEndpoint primary, secondary;
+  google::scp::cpio::PrivateKeyVendingEndpoint primary;
```

Comment:
This file includes changes included the previous PR https://github.com/privacysandbox/bidding-auction-servers/pull/7.
They are necessary here to test the other changes.